tasks/CentOS/main.yml

```diff
@@ -15,7 +15,10 @@
       description: The Sensu Core yum repository
       baseurl: "{{ sensu_yum_repo_url }}"
       gpgkey: "{{ sensu_yum_key_url }}"
-      gpgcheck: yes
+      gpgcheck: "{{ (
```

Comment:
@smbambling - Is the reasoning for this because the packages for CentOS 5/REHL 5 are not signed? Or does the signing simply not work?

Reply:
They are not signed and per the sensu devs there are technical issues that limit them from signing the older RPMs. 

I figured this was a good compromise that forces security where it can be used and disables it when it can’t. Plus these systems are EOL these unsigned RPMs seems to be a small concern in the big picture

Reply:
Yeah, this seems totally sane to me.